app/views/users/couches/new.html.haml

```diff
@@ -1,28 +1,30 @@
-%h1 Add Couch
-%br
-= form_for [@user, @couch] do |f|
-  = f.label :name
-  %br
-  = f.text_field :name
-  %br
-  = f.label :description
-  %br
-  = f.text_area :description
-  %br
-  = f.label :street_address
-  %br
-  = f.text_area :street_address
-  %br
-  = f.label :city
-  %br
-  = f.text_area :city
-  %br
-  = f.label :state
-  %br
-  = f.text_area :state
-  %br
-  = f.label :zipcode
-  %br
-  = f.text_area :zipcode
-  %br
-  = f.submit "Add Couch"
\ No newline at end of file
+%section.vanilla_form
+  %h1 Add Couch
+  %br
+  %article
+    = form_for [@user, @couch] do |f|
+      = f.label :name
+      %br
```

Comment:
@ski-climb @njgheorghita we can also keep this in mind on the next styling shift.